services/src/main/java/org/exoplatform/news/rest/NewsRestResourcesV1.java

```diff
@@ -173,6 +173,9 @@ public Response scheduleNews(@Context HttpServletRequest request,
       if (news == null) {
         return Response.status(Response.Status.NOT_FOUND).build();
       }
+      if (!Boolean.TRUE.equals(news.isCanEdit())) {
```

Comment:
Could we improve this ? cc @azayati 
```suggestion
      if (!Boolean.TRUE.equals(news.isCanEdit())) {
```
```suggestion
      if (Boolean.FALSE.equals(news.isCanEdit())) {
```
or 
```suggestion
      if (!news.isCanEdit()) {
```
or 
```suggestion
      if (news.isCanEdit() == false) {
```

Reply:
it was !news.isCanEdit(), but we have changed as recommended by Sonar Check

Reply:
cc @hassenelkamel-exo